app/Console/Commands/GenerateSitemap.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Carbon\Carbon;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\Route;
+
+class GenerateSitemap extends Command
+{
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $signature = 'sitemap:generate';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Generate the sitemap.';
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $routes = ['/' => ['en' => 'en', 'asl' => 'asl', 'fr' => 'fr', 'lsq' => 'lsq']];
+        // once deployed to the server, files have the same modified date, use README as a default last modified date
+        $default_lastmod = Carbon::createFromTimestamp(filemtime('./README.md'))->toISOString();
+        foreach (Route::getRoutes()->get('GET') as $route) {
+            if (str_contains($route->getName(), 'about')) {
```

Comment:
I still think using `str_contains` is susceptible to false positives. How about using [`$route->named()`](https://laravel.com/api/10.x/Illuminate/Routing/Route.html#method_named). This takes in patterns that can be matched and returns a boolean value. So something like `$route->named('*.about.*')`. I think it might be better though to define this in a config file as an array, then you'd call something like `$route->named(config('sitemap.patterns'))` to test against the routes. You could even be more explicit and use the specific locales instead of the first wildcard if you'd like.

Reply:
Thanks Justin, I forgot to update this part, I agree with you about str_contains can be false positive. I added a new config file called seo.php and took the approach you suggested.